platform/keyword-registry/src/main/java/org/codice/ddf/keyword/KeywordRegistryImpl.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.keyword;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class KeywordRegistryImpl implements KeywordRegistry {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(KeywordRegistryImpl.class);
+
+  private List<String> keywordList = Collections.emptyList();
+
+  private Map<String, String> keywordMap = Collections.emptyMap();
+
+  @Override
+  public Optional<String> getKeyword(String keyword) {
+    return Optional.ofNullable(keywordMap.get(keyword));
+  }
+
+  @Override
+  public Map<String, String> getKeywords() {
+    return keywordMap;
+  }
+
+  @SuppressWarnings("WeakerAccess" /* public method needed for blueprint/metatype */)
+  public void setKeywordList(List<String> keywordList) {
+    this.keywordMap = new HashMap<>();
+    this.keywordList = new ArrayList<>();
+
+    LOGGER.info("Setting up keyword registry with values from {}", keywordList);
+
+    for (String keyword : keywordList) {
+      String[] parts = keyword.split("=");
+
+      if (parts.length == 2
+          && StringUtils.isNotEmpty(parts[0])
+          && StringUtils.isNotEmpty(parts[1])) {
```

Comment:
I would use `StringUtils.isNotBlank`, which checks if a String is not empty (""), not null and not whitespace only, whereas `StringUtils.isNotEmpty` only checks if a String is not empty ("") and not null. Because as is, a space would pass the if but an empty String would be added to `keywordMap` on line 59